docs/mesos/RpmCreateInstallCalico.md

```diff
@@ -0,0 +1,28 @@
+# Create and Install the Calico Mesos Plugin RPM
+This guide will walk you through building and installing the Calico-Mesos plugin from an RPM that can be built from the [calico-mesos repository](https://github.com/projectcalico/calico-mesos). The RPM will automate the installation of everything needed to run Calico in your Mesos cluster. 
+
+The Calico-Mesos plugin must be installed with Calico's core services running on every Agent node in your cluster in order to take advantage of Calico networking. 
+
+*This guide assumes you already have Mesos + Net-Modules installed on your Agent(s).* 
+
+# Build and Install the RPM
+The rpm can be built from the calico-mesos repository by calling `make rpm` calico-mesos directory as shown below. Once the rpm is created, install the package.
+
+    $ git clone https://github.com/projectcalico/calico-mesos.git
+    $ cd calico-mesos
+    $ make rpm
+    $ sudo yum install dist/calico-mesos-0.1.1-1.el7.centos.x86_64.rpm
+
+# Start Calico Services
+A systemd unit file has been provided to start the Calico processes needed by the calico_mesos plugin. When starting the calico-mesos service, the environment variable `ETCD_AUTHORITY` is used to point Calico to a running instance of etcd. This variable is set in `/etc/default/mesos-slave`. Open this file and ensure that the `ETCD_AUTHORITY` variable is set correctly, then run the following commands.
+
```

Comment:
Add a note linking to the External Services guide for setting up etcd (if you haven't done so already, yadda yadda...)